octobot/src/server/github_handler.rs

```diff
@@ -1023,4 +1055,12 @@ impl GithubEventHandler {
         );
         self.pr_merge.send(req);
     }
+
+    fn build_thread_guid(&self, repo: Repo, number: u32) -> String {
```

Comment:
```suggestion
    fn build_thread_guid(&self, repo: &Repo, number: u32) -> String {
```

should take repo by reference, not require a copy/move